### DISCUSSION TITLE
2FA makes publishing from continuous deployment difficult

### DISCUSSION THREAD

**Discussion by MylesBorins** [npm/feedback#8] [Registry] [answered]
**Problem Statement**

If an account enables 2FA it is no longer simple to publish via automation such as a github action. 

**Potential Solution**

There are some community solution such as [wombat dressing room](https://github.com/GoogleCloudPlatform/wombat-dressing-room)... but it would be much nicer if there was an official solution to make automated publishing easier while not reducing the security of accounts with 2FA enabled.

**Comment by wesleytodd** [comment, marked as answer]
Should this be closed as a duplicate of https://github.com/npm/feedback/discussions/44?

edit (@MylesBorins): the original link was to #46 which is not about 2Fa

**Reply by MylesBorins** [reply]
There is not really a "closed" in discussions. But let's considered this answered as a pointer to #44
